src/consul/core.clj

```diff
@@ -437,7 +435,8 @@
   ([conn]
    (catalog-datacenters conn {}))
   ([conn {:as params}]
-   (consul-index conn :get [:catalog :datacenters] {:query-params params})))
+   (-> (consul-index conn :get [:catalog :datacenters] {:query-params params})
+       :body)))
```

Comment:
From the PR description it is not clear why this is done...

Reply:
I will extend the explanation.